WORKSPACE

```diff
@@ -79,6 +79,35 @@ git_repository(
     tag = "5.3.0-21.7",
 )
 
+# We use `perf_data_converter` with a patch to make it more usable as a library.
+git_repository(
+    name = "com_google_perf_data_converter",
+    commit = "442981cd4071fa9b1057b2609406db027e6d6263",
+    patch_args = ["-p1"],
+    patch_cmds = [
+        "mv ./src/quipper .",
+        "rm -rf ./src",
+    ],
```

Comment:
It does the same thing, but using `patch_cmds` is a bit more convenient since the commands are applied after the patch is, which makes it easier to generate the patch. On the other hand, IIUC, `strip_prefix` is applied before the patch so the diff needs to be from after the `mv` rather than from a fresh clone.